tests/integration/override_config_params/zed/test_override_config_params.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from jina import Flow, Document
+from tests import validate_callback
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def test_override_config_params(mocker):
+    def validate_response(resp):
+        doc = resp.docs[0]
+        assert doc.tags['param1'] == 50
+        assert doc.tags['param2'] == 30
+        assert doc.tags['param3'] == 10  # not overriden
+        assert doc.tags['name'] == 'name'  # not override
+        assert doc.tags['workspace'] == 'different_workspace'
+
+    mock = mocker.Mock()
```

Comment:
if it's not too much of a bother, can you refactor to use `return_results` instead?

Reply:
Done!